### PR TITLE
Enhancement/Datepicker for Web

### PR DESCRIPTION
**Pull request by carelulu-kenneth**
**Summary**
* Updated css for datepicker for web

This PR fixes/implements the following **bugs/features**
* [ ] Feature uses month and year dropdown

**Test plan (required)**
time-final
date-final

**Closing issues**
closes https://github.com/CareLuLu/v3-app/issues/1427

### CODE REVIEW

File: src/Datepicker/index.web.js [hunk 1]
```diff
@@ -269,6 +288,8 @@ const Datepicker = (props) => {
         style={currentStyle}
         strictParsing
         autoComplete={autoComplete}
+        showMonthDropdown
+        showYearDropdown
```
Comment (carelulu-kevin) [inline src/Datepicker/index.web.js:292]: Let's allow this prop `showMonthDropdown` and `showYearDropdown` to be configurable. Add this to https://github.com/CareLuLu/react-native-web-ui-components/blob/7d459f8348f6a3f6e33539ba91018a6cb3f019dc/src/Datepicker/index.web.js#L298 and set default to `true` https://github.com/CareLuLu/react-native-web-ui-components/blob/7d459f8348f6a3f6e33539ba91018a6cb3f019dc/src/Datepicker/index.web.js#L318.

**Review by carelulu-kevin — COMMENTED** [review]
Thanks for working on this @carelulu-kenneth. Please see my comment.

**Review by CareLuLu-Gabriel — APPROVED** [review]
Great job, @carelulu-kenneth! LGTM!

**Review by carelulu-igor — APPROVED** [review]
Good job @carelulu-kenneth!

**Review by carelulu-kevin — CHANGES_REQUESTED** [review]
@carelulu-kenneth please check this part of the css. currently, it's not visible with a white background.

```css
.react-datepicker__day{cu.react-datepicker__navigation--nextnd-color:#3dcc4a;color:#fff}
```

Screen Shot 2021-06-03 at 12 22 11 AM

Screen Shot 2021-06-03 at 12 22 21 AM

**Pull request merged**